client/src/cmdhfmf.c

```diff
@@ -4261,10 +4271,11 @@ static int CmdHF14AMfCSetUID(const char *Cmd) {
 static int CmdHF14AMfCWipe(const char *cmd) {
     CLIParserContext *ctx;
     CLIParserInit(&ctx, "hf mf cwipe",
-                  "Wipe gen1 magic chinese card.\n"
-                  "Set UID / ATQA / SAK / Data / Keys / Access to default values",
+                  "Wipe a gen1 magic card.\n"
```

Comment:
Consistency with  `Gen1a` vs `gen1`    all across this pr in term of Capital letter or not.